.github/workflows/rust.yml

```
@@ -6,130 +6,74 @@ on:
         branches:
             - main
 
+env:
+  CARGO_INCREMENTAL: 0
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)